src/main/java/com/aws/greengrass/shadowmanager/ShadowManagerDatabase.java

```diff
@@ -41,6 +41,7 @@ public class ShadowManagerDatabase implements Closeable {
             + ";RETENTION_TIME=1000" // ms - time to keep values for before writing to disk (default is 45000)
             + ";DEFRAG_ALWAYS=TRUE" // defragment db on shutdown (ensures only a single value in db on close)
             + ";COMPRESS=TRUE" // compress large objects (clob/blob) (default false)
+            + ";WRITE_DELAY=0" // maximum delay between commit and flushing the log
```

Comment:
what's the difference with retention time? They seem really similar.

Reply:
and if we have this, do we also need the sql command?

Reply:
The way I understood is that CHECKPOINT SYNC does more than just flushing the commit to log and that it flushes data and other index files as well.

Reply:
IMO, if we're writing the log immediately, they we can let go of retention time as we no longer need it. If I remove it, it is set to default value of 45 seconds. Not confident about making it 0. Hence left it to what it was before.